doc/sphinx/practical-tools/coq-commands.rst

```diff
@@ -97,8 +90,8 @@ In order of importance they are:
 `coqrc` start up script
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-When Coq is launched, it can implicitly prepend a startup script to any document
-read by Coq, whether it is an interactive session or a file to compile.
+When Rocq is launched, it can implicitly prepend a startup script to any document
+read by Rocq, whether it is an interactive session or a file to compile.
```

Comment:
```suggestion
When Rocq is launched, it can implicitly prepend a startup script to any document
it reads, whether it is an interactive session or a file to compile.
```